// @vue-macros/common@1.0.0/dist/ast.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});














var _chunkZ3AMPVRMjs = require('./chunk-Z3AMPVRM.js');
require('./chunk-5WJ4WTPA.js');
require('./chunk-Z2XCFJPE.js');















exports.TS_NODE_TYPES = _chunkZ3AMPVRMjs.TS_NODE_TYPES; exports.babelParse = _chunkZ3AMPVRMjs.babelParse; exports.checkInvalidScopeReference = _chunkZ3AMPVRMjs.checkInvalidScopeReference; exports.isCallOf = _chunkZ3AMPVRMjs.isCallOf; exports.isFunctionType = _chunkZ3AMPVRMjs.isFunctionType; exports.isLiteralType = _chunkZ3AMPVRMjs.isLiteralType; exports.isStaticExpression = _chunkZ3AMPVRMjs.isStaticExpression; exports.isStaticObjectKey = _chunkZ3AMPVRMjs.isStaticObjectKey; exports.resolveLiteral = _chunkZ3AMPVRMjs.resolveLiteral; exports.resolveObjectExpression = _chunkZ3AMPVRMjs.resolveObjectExpression; exports.resolveObjectKey = _chunkZ3AMPVRMjs.resolveObjectKey; exports.resolveTemplateLiteral = _chunkZ3AMPVRMjs.resolveTemplateLiteral; exports.unwrapTSNode = _chunkZ3AMPVRMjs.unwrapTSNode; exports.walkAST = _chunkZ3AMPVRMjs.walkAST;
exports.default = module.exports;
