// @vue-macros/common@1.0.0/dist/chunk-O43UNDSZ.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true}); function _nullishCoalesce(lhs, rhsFn) { if (lhs != null) { return lhs; } else { return rhsFn(); } }

var _chunkZ3AMPVRMjs = require('./chunk-Z3AMPVRM.js');


var _chunk5WJ4WTPAjs = require('./chunk-5WJ4WTPA.js');


var _chunkZ2XCFJPEjs = require('./chunk-Z2XCFJPE.js');

// src/vue.ts
var _compilersfc = require('@vue/compiler-sfc');
function parseSFC(code, id) {
  var _a;
  const sfc = _compilersfc.parse.call(void 0, code, {
    filename: id
  });
  const { descriptor, errors } = sfc;
  const lang = (_a = descriptor.script || descriptor.scriptSetup) == null ? void 0 : _a.lang;
  return {
    sfc,
    ...descriptor,
    lang,
    errors,
    getSetupAst() {
      if (!descriptor.scriptSetup)
        return;
      return _chunkZ3AMPVRMjs.babelParse.call(void 0, descriptor.scriptSetup.content, lang);
    },
    getScriptAst() {
      if (!descriptor.script)
        return;
      return _chunkZ3AMPVRMjs.babelParse.call(void 0, descriptor.script.content, lang);
    }
  };
}
function getFileCodeAndLang(code, id) {
  var _a;
  if (!_chunkZ2XCFJPEjs.REGEX_VUE_SFC.test(id))
    return { code, lang: _chunk5WJ4WTPAjs.getLang.call(void 0, id) };
  const sfc = parseSFC(code, id);
  const scriptCode = _nullishCoalesce(((_a = sfc.script) == null ? void 0 : _a.content), () => ( ""));
  return {
    code: sfc.scriptSetup ? `${scriptCode}
;
${sfc.scriptSetup.content}` : scriptCode,
    lang: _nullishCoalesce(sfc.lang, () => ( "js"))
  };
}
function addNormalScript({ script, lang }, s) {
  return {
    start() {
      if (script)
        return script.loc.end.offset;
      const attrs = lang ? ` lang="${lang}"` : "";
      s.prependLeft(0, `<script${attrs}>`);
      return 0;
    },
    end() {
      if (!script)
        s.appendRight(0, `
<\/script>
`);
    }
  };
}





exports.parseSFC = parseSFC; exports.getFileCodeAndLang = getFileCodeAndLang; exports.addNormalScript = addNormalScript;
