// @vue-macros/common@1.0.0/dist/constants.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});



















var _chunkZ2XCFJPEjs = require('./chunk-Z2XCFJPE.js');




















exports.DEFINE_EMITS = _chunkZ2XCFJPEjs.DEFINE_EMITS; exports.DEFINE_MODEL = _chunkZ2XCFJPEjs.DEFINE_MODEL; exports.DEFINE_MODEL_DOLLAR = _chunkZ2XCFJPEjs.DEFINE_MODEL_DOLLAR; exports.DEFINE_OPTIONS = _chunkZ2XCFJPEjs.DEFINE_OPTIONS; exports.DEFINE_PROPS = _chunkZ2XCFJPEjs.DEFINE_PROPS; exports.DEFINE_PROPS_DOLLAR = _chunkZ2XCFJPEjs.DEFINE_PROPS_DOLLAR; exports.DEFINE_PROPS_REFS = _chunkZ2XCFJPEjs.DEFINE_PROPS_REFS; exports.DEFINE_RENDER = _chunkZ2XCFJPEjs.DEFINE_RENDER; exports.DEFINE_SETUP_COMPONENT = _chunkZ2XCFJPEjs.DEFINE_SETUP_COMPONENT; exports.DEFINE_SLOTS = _chunkZ2XCFJPEjs.DEFINE_SLOTS; exports.REGEX_LANG_JSX = _chunkZ2XCFJPEjs.REGEX_LANG_JSX; exports.REGEX_LANG_TS = _chunkZ2XCFJPEjs.REGEX_LANG_TS; exports.REGEX_NODE_MODULES = _chunkZ2XCFJPEjs.REGEX_NODE_MODULES; exports.REGEX_SETUP_SFC = _chunkZ2XCFJPEjs.REGEX_SETUP_SFC; exports.REGEX_SRC_FILE = _chunkZ2XCFJPEjs.REGEX_SRC_FILE; exports.REGEX_VUE_SFC = _chunkZ2XCFJPEjs.REGEX_VUE_SFC; exports.REGEX_VUE_SUB = _chunkZ2XCFJPEjs.REGEX_VUE_SUB; exports.REPO_ISSUE_URL = _chunkZ2XCFJPEjs.REPO_ISSUE_URL; exports.WITH_DEFAULTS = _chunkZ2XCFJPEjs.WITH_DEFAULTS;
exports.default = module.exports;
